sunpy/util/util.py

```diff
@@ -97,7 +98,7 @@ def expand_list(inp):
 
 def expand_list_generator(inp):
     for item in inp:
-        if isinstance(item, (tuple, list, UserList)):
+        if not isinstance(item, str) and isinstance(item, Iterable):
```

Comment:
> While I doubt someone will pass a byte string in the context of this MR, as this is a utility function, I believe you want to avoid an issue with byte strings. In the current implementation a byte string wouldn't be caught by `isinstance(item, str)` as a regular string or raw string literal would be.

As suggested by @PaulJWright